packages/react/src/components/Tag/InteractiveTag.stories.js

```diff
@@ -0,0 +1,305 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState } from 'react';
+import { default as Tag } from '.';
+import { default as SelectableTag } from './SelectableTag';
+import { default as OperationalTag } from './OperationalTag';
+import { default as DismissibleTag } from './DismissibleTag';
+import { Asleep } from '@carbon/icons-react';
+import { Tooltip } from '../Tooltip';
+import { Toggletip, ToggletipButton, ToggletipContent } from '../Toggletip';
+import { Popover, PopoverContent } from '../Popover';
+import mdx from './InteractiveTag.mdx';
+
+export default {
+  title: 'Experimental/unstable__InteractiveTag',
+  component: SelectableTag,
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+export const Selectable = () => {
+  return (
+    <>
+      <SelectableTag className="some-class" title="Clear Filter">
```

Comment:
It seems like `SelectableTag` and `OperationalTag` have a `title` prop that doesn't seem to do anything, these can probably be removed right?